### DISCUSSION TITLE
How can I download install.sh from v2.12.1?

### DISCUSSION THREAD

**Discussion by pedroherpeto** [chatwoot/chatwoot#6305] [Q&A] [answered]
Hi everyone, how can I download install.sh from v2.12.1?

thanks

**Comment by vishnu-narayanan** [comment]
@pedroherpeto I don't follow. Could you please rephrase?

**Reply by pedroherpeto** [reply]
hi @vishnu-narayanan ... yes.. how can i get a previous version of it: https://get.chatwoot.app/linux/install.sh ?
thank you

**Reply by vishnu-narayanan** [reply]
@pedroherpeto By default, the installer script will always try to install the latest version.

**Comment by cahflitto** [comment]
We think, version v2.12.1 is stable and flexible to configure, maybe our point is, can we use that version specific?

**Comment by vishnu-narayanan** [comment, marked as answer]
By default, the Linux installer(https://get.chatwoot.app/linux/install.sh) is set to get the latest Chatwoot version.

To install a specific version, could you try

```
./install.sh --Install <branch/tag-name>
```

For example,

```
./install.sh --Install v2.12.1
```

**Reply by vishnu-narayanan** [reply]
cc: @pedroherpeto @cahflitto

**Reply by cahflitto** [reply]
thanks @vishnu-narayanan it's going well, I appreciate it
